shared/AppInsightsCore/src/JavaScriptSDK/HelperFuncs.ts

```diff
@@ -268,35 +367,71 @@ export function isSymbol(value: any): boolean {
     return typeof value === "symbol";
 }
 
+/**
+ * Checks if the type of the value is a normal plain object (not a null or data)
+ * @param value
+ */
+export function isPlainObject(value: any): boolean {
```

Comment:
Leveraged most of this logic from jQuery to specifically identify ONLY objects and not other objects (arrays etc) that return a typeof value === "object"